packages/cli/src/commands/env/pull.ts

```diff
@@ -135,3 +152,79 @@ function escapeValue(value: string | undefined) {
         .replace(new RegExp('\r', 'g'), '\\r') // combine newlines (windows) into one line
     : '';
 }
+
+async function createEnvObject(
```

Comment:
These functions related to env diffing could be better organized into a separate file, in my opinion.